config/sources/odroidxu4.conf

```diff
@@ -57,6 +57,10 @@ write_uboot_platform()
 		dd if=$1/bl2.bin.hardkernel.1mb_uboot of=$2 seek=$bl2_position conv=fsync >/dev/null 2>&1
 		dd if=$1/u-boot-dtb.bin of=$2 bs=512 seek=$uboot_position conv=fsync >/dev/null 2>&1
 		dd if=$1/tzsw.bin.hardkernel of=$2 seek=$tzsw_position conv=fsync >/dev/null 2>&1
+
```

Comment:
is it useful & even working? I can only find the files in the image in /usr/lib/linux-u-boot-next-odroidxu4_5.21-GRO_armhf